.github/workflows/docs-ci.yaml

```diff
@@ -86,5 +85,17 @@ jobs:
       - run: make ${{ inputs.make-target }}
         working-directory: ${{ inputs.docs-directory }}
 
-      - run: make linkcheck
+  linkcheck:
+    if: inputs.pip-install-target != ''
```

Comment:
This prevents linkcheck from running on conda-based build environments. It should be split into `linkcheck-conda` and `linkcheck-pip`.

Another approach would be what @tsibley proposed in https://github.com/nextstrain/.github/pull/28#pullrequestreview-1173000933. This would allow for a single job to handle both conda and pip, but it's more complex:

> My idea had been to make this workflow read the RTD config file and install what's configured there, which I still think would be nicer, but it's more complex.

Reply:
> This prevents linkcheck from running on conda-based build environments. It should be split into `linkcheck-conda` and `linkcheck-pip`.

I don't see the utility of running the exact same linkcheck process twice — that's just going to cause additional fails due to random network glitches.

In what scenario does a link resolve in the conda build and fail to resolve in the pip build, or vice versa?

Reply:
`build-conda` and `build-pip` are meant to be mutually exclusive. This is not checked in the workflow because it's reasonable to assume that a docs project will use only one. See GitHub query [`org:nextstrain nextstrain/.github/.github/workflows/docs-ci.yaml`](https://github.com/search?q=org%3Anextstrain+nextstrain%2F.github%2F.github%2Fworkflows%2Fdocs-ci.yaml&type=code): some invocations specify `environment-file` while others specify `pip-install-target`.